lms/static/scripts/frontend_apps/services/content-info-fetcher.js

```diff
@@ -55,13 +75,17 @@ export class ContentInfoFetcher {
       },
       item: {
         title: metadata.item.title,
-
-        // TODO - Fill in this information from the API
-        containerTitle: '',
+        subtitle: metadata.item.subtitle,
+        // TODO Remove once client updated to look this up in the
+        // `container` object
+        containerTitle: metadata.container.title,
+      },
+      container: {
+        title: metadata.container.title,
+        subtitle: metadata.container.subtitle,
       },
```

Comment:
Is it safe to do things like `container: metadata.container` here?

Reply:
It's certainly possible to do that reference assignment, but the benefits of how it's done here are:

* Complete context of the object shape being passed along to the client, for future devs human-parsing this code — i.e. you can hold the whole context in your head
* More "guarantee" that `container` is in fact an object — this code will, in effect, throw if `metadata.container` is not an object when we try to look up properties on it, and that's intentional/desirable
* If the shape of the returned `container` object is augmented, we're not blindly passing on unexpected properties. This wouldn't necessarily be harmful, but it's ideal to be explicit here.